rakelib/test.rake

```diff
@@ -6,9 +6,14 @@ require 'pathname'
 
 namespace "test" do
 
-  desc "run the java tests"
+  desc "run the java unit tests"
   task "core-java" do
-    exit(1) unless system './gradlew clean test --info'
+    exit(1) unless system './gradlew clean test'
+  end
+
+  desc "run the ruby unit tests"
+  task "core-ruby" do
+    exit 1 unless system(*default_spec_command)
```

Comment:
This should probably be `exec(*default_spec_command)` which simply replaces the current process with the executed command. You won't need to explicitly `exit` in that case since `exec` replaces the running process.

Reply:
hmm... i am going to leave this one as-is.  This pattern is copied from many other locations in our code (and same file), and apprehensive of changing it right now.

Reply:
Works for me. The efficiency of `exec` isn't realized on jruby anyway, since there's no native way of doing a real process image replacement in java.